configure.ac

```
@@ -817,7 +819,7 @@ natdynlinkopts=""
 AS_IF([test x"$enable_shared" != "xno"],
   [AS_CASE([$host],
     [*-apple-darwin*],
-      [mksharedlib="$CC -shared -flat_namespace -undefined suppress \
+      [mksharedlib="$CC -shared $(LDFLAGS) -flat_namespace -undefined suppress \
```

This and the other three either need to be:
```suggestion
      [mksharedlib="$CC -shared \$(LDFLAGS) -flat_namespace -undefined suppress \
```
or:
```suggestion
      [mksharedlib="$CC -shared $LDFLAGS -flat_namespace -undefined suppress \
```
The first one reads `LDFLAGS` when `make` is run, the second reads `LDFLAGS` at `configure`-time. I'm actually not sure which one you want for this - the current one means run a command called `LDFLAGS` which is causing error messages in `configure`.

---

I went for the first option which is more consistent with how other dependencies are treated.